Articles/ShortArticles/TheDeveloperCertificateOfOrigin.md

```diff
@@ -56,7 +56,7 @@ which implies that the developer is asserting the terms of the DCO<sup>[7]</sup>
 
 This `Signed-of-by` line can be added to the Git commit log automatically using the standard `-s` option with `git commit -s` (using information already registered with Git locally).
 
-Some major projects are using the DCO with Git commit sign-offs for handling contributions including the Linux Kernel<sup>[8]</sup> and more recently the source code for GitLab itself <sup>[3],[4]</sup>.
+Some major projects are using the DCO with Git commit sign-offs for handling contributions, including the Linux Kernel<sup>[8]</sup> and more recently, the source code for GitLab itself <sup>[3],[4]</sup>.
```

Comment:
I am not sure the comma  at "... including the Linux Kernel[8] and more recently, the source code for GitLab ..." makes sense.  If there is going to be a comma there, then there should also be a comma before the "and" as well, no?  So this would be:

"Some major projects are using the DCO with Git commit sign-offs for handling contributions, including the Linux Kernel<sup>[8]</sup>, and more recently, the source code for GitLab itself <sup>[3],[4]</sup>."

?

Reply:
Good catch here! I believe that the second comma would actually go after the "and", as we're breaking up the interjection of "more recently". So, it would look like:

"Some major projects are using the DCO with Git commit sign-offs for handling contributions, including the Linux Kernel[8] and **, more recently,** the source code for GitLab itself [3],[4]."

Reply:
Thanks! Should be resolved in my last commit 9ec2820627836ebe79c37b588db6556142b06f83.